cana/boolean_node.py

```diff
@@ -636,6 +636,43 @@ def bias(self):
 		"""
 		return sum(self.outputs) / 2**self.k
 
-
-
-
+	def c_sensitivity(self, c, mode="default", max_k=0):
+		"""give the c-sensitivity of this node's Boolean function
```

Comment:
I'm not sure. I'm new to this "pull request" thing. But I guess we will learn by trial-and-error ;)